src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -59,16 +60,22 @@ class SearchFilter extends AbstractContextAwareFilter
 
     protected $iriConverter;
     protected $propertyAccessor;
+    protected $identifiersExtractor;
 
     /**
      * @param RequestStack|null $requestStack No prefix to prevent autowiring of this deprecated property
      */
-    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, IriConverterInterface $iriConverter, PropertyAccessorInterface $propertyAccessor = null, LoggerInterface $logger = null, array $properties = null)
+    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, IriConverterInterface $iriConverter, PropertyAccessorInterface $propertyAccessor = null, LoggerInterface $logger = null, array $properties = null, IdentifiersExtractorInterface $identifiersExtractor = null)
     {
         parent::__construct($managerRegistry, $requestStack, $logger, $properties);
 
+        if (null === $identifiersExtractor) {
+            @trigger_error('Not injecting ItemIdentifiersExtractor is deprecated since API Platform 2.3 and will not be possible anymore in API Platform 3.0.', E_USER_DEPRECATED);
+        }
```

Comment:
Shouldn't we say that not having identifiers extractor leads to bugs?

Reply:
It leads to the existing behaviour, we can add it in the message.